assets/src/scripts/contributors.js

```diff
@@ -79,6 +79,8 @@ jQuery( function ( $ ) {
 			imageHeight: realHeight,
 			minWidth: xImg > xInit ? xInit : xImg,
 			minHeight: yImg > yInit ? yInit : yImg,
+			maxHeight: minPictureSize * 2,
+			maxWidth: minPictureSize * 2,
```

Comment:
@SteelWagstaff I defined in this PR the maximum allowed size for a cropped image to 800x800, which means at the moment to crop a contributor's picture, the maximum size allowed to cover is 800x800 px.
I can remove or change this limit, but I think keep it relatively small is better for performance purposes.